tests/installation/install_and_reboot.pm

```diff
@@ -149,6 +149,8 @@ sub run() {
         # we require ssh installation anyway
         if (check_var('BACKEND', 'ipmi')) {
             use_ssh_serial_console;
+            # set serial console for xen
+            &set_serial_console_on_xen("/mnt") if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen"));
```

Comment:
... because you actually defined the function with an empty prototype. Please remove all () - this is bash syntax. For perl it means: does not take arguments.